src/Evolution/Systems/GrMhd/ValenciaDivClean/NewmanHamlin.cpp

```diff
@@ -89,7 +94,10 @@ boost::optional<PrimitiveRecoveryData> NewmanHamlin::apply(
              (magnetic_field_squared + 2.0 * rho_h_w_squared)) /
         square(rho_h_w_squared * root_of_cubic);
 
-    ASSERT(0.0 <= v_squared and v_squared <= 1.0,
+    // If this fails, there was code in the Bitbucket version that adjusted
```

Comment:
Should this code comment be part of the assert? Not sure, I'll let you decide :)

Reply:
I'm going to leave it as a comment, as I'll be converting the ASSERT to returning a boost::none in a forthcoming PR